src/data/matrix/basic.lean

```diff
@@ -364,108 +377,96 @@ theorem diagonal_mul_diagonal' [decidable_eq n] (d₁ d₂ : n → α) :
 diagonal_mul_diagonal _ _
 
 @[simp]
-lemma map_mul {L : matrix m n α} {M : matrix n o α}
-  {β : Type w} [semiring β] {f : α →+* β} :
+lemma map_mul {L : matrix m n α} {M : matrix n o α} [semiring β] {f : α →+* β} :
   (L ⬝ M).map f = L.map f ⬝ M.map f :=
 by { ext, simp [mul_apply, ring_hom.map_sum], }
 
+lemma is_add_monoid_hom_mul_left (M : matrix l m α) :
```

Comment:
Do we want bundled versions of these?

Reply:
Maybe, but I'd argue that's out of scope - these lines are just moved from below

Reply:
No, it's fine.